packages/app/src/PasswordCheck.tsx

```diff
@@ -63,39 +68,25 @@ export const CheckOrX = ({
 const Check = () => (
   <svg
     xmlns="http://www.w3.org/2000/svg"
-    width="1em"
-    height="1em"
+    width="16"
```

Comment:
I wonder if we can use bootstrap icon directly without using this svg component (for example: https://github.com/hyperdxio/hyperdx/blob/2fcd167540f596f800b042c9403dfbcc3072fffe/packages/app/src/InstallInstructionsModal.tsx#L33-L38)

Reply:
Sure, will do, didn't know if you were using the font or not, will slot that in.